modules/distr-tracing-tempo-deploy-streaming.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
```

Comment:
Since for Tempo we don't have different deployment strategies... I think we can remove this file.

Reply:
Outdated. Resolving this thread.